Src/FluentAssertions/DataRowCollectionAssertionExtensions.cs

```diff
@@ -0,0 +1,347 @@
+        }
+
+        /// <summary>
+        /// Asserts that a <see cref="DataRowCollection"/> contains at least one <see cref="DataRow"/> equivalent to another <see cref="DataRow"/>.
+        /// </summary>
+        /// <remarks>
+        /// See the documentation for <see cref="DataRowAssertions{TDataRow}"/> for information about when <see cref="DataRow"/>
+        /// objects are considered equivalent.
+        /// </remarks>
+        /// <param name="expected">The expected <see cref="DataRow"/>.</param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <paramref name="because"/>.
+        /// </param>
+        public static AndConstraint<GenericCollectionAssertions<DataRow>> ContainEquivalentOf(this GenericCollectionAssertions<DataRow> assertion, DataRow expected, string because = "",
+            params object[] becauseArgs)
+        {
+            return ContainEquivalentOf(assertion, expected, config => config, because, becauseArgs);
+        }
+
+        /// <summary>
+        /// Asserts that a <see cref="DataRowCollection"/> contains at least one object equivalent to another object.
+        /// </summary>
+        /// <remarks>
+        /// See the documentation for <see cref="DataRowAssertions{TDataRow}"/> for information about when <see cref="DataRow"/>
+        /// objects are considered equivalent.
+        /// </remarks>
+        /// <param name="expected">The expected <see cref="DataRow"/>.</param>
+        /// <param name="config">
+        /// A reference to the <see cref="EquivalencyAssertionOptions{DataRow}"/> configuration object that can be used
+        /// to influence the way the object graphs are compared. You can also provide an alternative instance of the
+        /// <see cref="EquivalencyAssertionOptions{DataRow}"/> class. The global defaults are determined by the
+        /// <see cref="AssertionOptions"/> class.
+        /// </param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <paramref name="because"/>.
+        /// </param>
+        public static AndConstraint<GenericCollectionAssertions<DataRow>> ContainEquivalentOf(this GenericCollectionAssertions<DataRow> assertion, DataRow expected, Func<EquivalencyAssertionOptions<DataRow>,
```

Comment:
Could you re-use the existing `GenericCollectionAssertions.ContainEquivalentOf`?
That would save you a lot of tests to verify the almost exact copy of that code.

Reply:
I just compared them closely and determined that they _were in fact identical_. I've removed them. Also, though, I don't think that functionality had tests yet. I'm inclined to _add tests_ to make sure that `DataRow` equivalency can in fact be tested by the regular code path, even though there's nothing specific to the type any more.